packages/frontend/src/components/Explorer/ResultsCard/QuickCalculations.tsx

```diff
@@ -90,13 +93,36 @@ const getSqlForQuickCalculation = (
     quickCalculation: QuickCalculation,
     fieldReference: string,
     sorts: SortField[],
+    warehouseType: WarehouseTypes | undefined,
 ) => {
+    let fieldQuoteChar = '"';
```

Comment:
this logic should already be available in common, do we need to redefine the mapping between warehouses and quote chars?

Reply:
Unfortunately, this is only really available in `packages/warehouses` and then used mostly on the BE.
I would have to get the warehouse client through a method like: https://github.com/lightdash/lightdash/blob/main/packages/warehouses/src/warehouseClientFromCredentials.ts
But that isn't available yet for consumption on the `frontend`...

Like @rephus mentioned here: https://github.com/lightdash/lightdash/pull/9704#pullrequestreview-1990904779 it would require a bigger refactoring to get this working as expected: a centralised class/util that would do this job for us  in both `backend` and `frontend` packages.

We then should follow the suggestion to create a tech debt ticket ^ and I can also add a TODO here. LMK what you think...